src/test/java/seedu/guest/storage/JsonUserPrefsStorageTest.java

```diff
@@ -120,4 +121,11 @@ public void saveUserPrefs_allInOrder_success() throws DataConversionException, I
         assertEquals(original, readBack);
     }
 
+    @Test
+    public void getUserPrefsFilePath() {
+        Path pefsFilePath = testFolder.resolve("TempPrefs.json");
```

Comment:
I think there's a typo with `prefsFilePath` instead of `pefsFilePath`

Reply:
Yes there is. Changed accordingly.